api/python/PE/objects/signature/pySignerInfo.cpp

```diff
@@ -41,7 +41,10 @@ void create<SignerInfo>(py::module& m) {
         "Should be 1")
 
     .def_property_readonly("issuer",
-        &SignerInfo::issuer,
+        [] (const SignerInfo& object) {
+          const issuer_t& issuer = object.issuer();
+          return std::pair<py::object, std::vector<uint8_t>>{safe_string_converter(std::get<0>(issuer)), std::get<1>(issuer)};
+        },
         "Issuer and serial number",
         py::return_value_policy::reference)
```

Comment:
This policy is no longer valid as you return a "stack" object. Prefer ``return_value_policy::copy``

Reply:
Good catch, I'll make this change.  Thanks!